src/page-modules/departures/layout.tsx

```diff
@@ -91,6 +89,16 @@ function DeparturesLayout({
     setIsSearching(false);
   };
 
+  const onSubmitHandler: FormEventHandler<HTMLFormElement> = (e) => {
```

Comment:
Couldn't the doSearch just be passed as the submitHandler?

Reply:
Need prevent default, and other places calling doSearch doesn't have access to event which has preventdefault